main.py

```diff
@@ -948,6 +954,13 @@ def notify_switch_enabled(self, dpid):
         event = KytosEvent(name=name, content={'dpid': dpid})
         self.controller.buffers.app.put(event)
 
+    def notify_switch_links_status(self, switch, reason):
+        """Send an event to notify the status of a link in a switch"""
+        with self._links_lock:
+            for link in self.links.values():
+                if switch in (link.endpoint_a.switch, link.endpoint_b.switch):
+                    self.notify_link_up_if_status(link, reason)
```

Comment:
@Alopalao good call consolidating this `notify_switch_links_status` method. `self.notify_link_up_if_status` is only expected to be called for when handling link going up or link being enabled, so please also remove that `and reason != "link disable"` conditional there. 

You could add a new conditional here to call `self.notify_link_status_change(self, link, reason=reason)`  instead when reason is `"link_disabled"`.  In the future, when we also support only notifying down again within a time interval then we can evolve this part here easily to call a new method.

Reply:
Ok, that sounds better.